nifi-nar-bundles/nifi-elasticsearch-bundle/nifi-elasticsearch-processors/pom.xml

```diff
@@ -56,6 +56,11 @@ language governing permissions and limitations under the License. -->
             <version>${es.version}</version>
         </dependency>
         <dependency>
+            <groupId>com.squareup.okhttp</groupId>
+            <artifactId>okhttp</artifactId>
+            <version>2.7.1</version>
```

Comment:
Mostly to be able to borrow code from InvokeHttp, and in the hopes that we might have a single "common" NAR with oft-used libraries someday, rather than multiple versions of OkHttp (etc.) in various NARs. I will update the version (and the code that uses the API) to latest